storage/engine/in_mem.go

```diff
@@ -28,7 +28,9 @@ type InMem struct {
 
 // NewInMem allocates and returns a new, opened InMem engine.
 func NewInMem(attrs roachpb.Attributes, cacheSize int64, stopper *stop.Stopper) InMem {
-	cache := NewRocksDBCache(cacheSize)
+	cache := NewRocksDBCache(cacheSize) // released when db closes
```

Comment:
Done.

On Mon, Jul 25, 2016 at 11:37 AM Tamir Duberstein <notifications@github.com>
wrote:

> In storage/engine/in_mem.go
> <https://github.com/cockroachdb/cockroach/pull/8002#discussion_r72087376>:
>
> > @@ -28,7 +28,9 @@ type InMem struct {
> >
> >  // NewInMem allocates and returns a new, opened InMem engine.
> >  func NewInMem(attrs roachpb.Attributes, cacheSize int64, stopper *stop.Stopper) InMem {
> > -	cache := NewRocksDBCache(cacheSize)
> > +	cache := NewRocksDBCache(cacheSize) // released when db closes
>
> "released when db closes" is not quite right - i'd remove this inline
> comment and only keep the paragraph below
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/8002/files/a8a18062dfd34f66cacf1ff4850143cba6bf14e9#r72087376>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AE135AGW8VG12j4nEpHZ2a7wNLid3lXQks5qZNgbgaJpZM4JT9du>
> .
>
-- 

-- Tobias